### PR TITLE
Feature/message list view

### PR DESCRIPTION
**Pull request by adrian09h**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: library/src/main/java/com/getstream/sdk/chat/view/MessageListView.java [hunk 1]
```diff
@@ -160,50 +161,65 @@ public Drawable getDrawableForAttachment(Message message, Boolean mine, List<Mes
...
                     strokeWidth = style.getMessageStrokeWidthMine();
-                    if (!isDefaultBubble()){
-                        topLeftRadius = style.getMessageTopLeftCornerRadiusMine();
-                        topRightRadius = style.getMessageTopRightCornerRadiusMine();
-                        bottomRightRadius = style.getMessageBottomRightCornerRadiusMine();
-                        bottomLeftRadius = style.getMessageBottomLeftCornerRadiusMine();
-                    }else{
+                    if (isDefaultBubble()) {
+                        try {
+                            if (message.getAttachments() != null
```
Comment (tschellenbach) [inline library/src/main/java/com/getstream/sdk/chat/view/MessageListView.java]: we do this often in our code. something like message.hasAttachments() would be nicer (checking both for null or zero size)

Comment (adrian09h) [reply library/src/main/java/com/getstream/sdk/chat/view/MessageListView.java]: Good idea.

File: sample/src/main/res/layout/activity_channel.xml [hunk 2]
```diff
@@ -33,20 +32,26 @@
...
-            stream:showUsersReactionDlg="true"
-            stream:userAvatarTextColor="#FFF"
-            stream:userAvatarTextStyle="bold"
-            stream:userReadStateAvatarHeight="15dp"
-            stream:userReadStateAvatarWidth="15dp"
-            stream:userReadStateTextColor="#FFF"
-            stream:userReadStateTextStyle="bold"
-            stream:userRreadStateTextSize="9sp" />
-
+            app:messageBackgroundColorMine="#FF9800"
```
Comment (tschellenbach) [inline sample/src/main/res/layout/activity_channel.xml]: shouldn't we use stream instead of app here?

Comment (tschellenbach) [reply sample/src/main/res/layout/activity_channel.xml]: The algolia library for instance uses the algolia prefix.

**Pull request merged**